modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/AccountRestrictionContainer.tsx

```diff
@@ -87,6 +88,9 @@ export function AccountRestrictionContainer({
 			if (isApproved && values.accountEntryRestrictedObjectFieldName) {
 				setDisableAccountSelect(true);
 			}
+			if (reload) {
+				setTimeout(() => setReload(false), 500);
+			}
```

Comment:
Hey, instead of creating a reload state and using a time out. We can just condition the rendering of the Clay Picker using the account restriction toggle.

For example:

https://github.com/liferay-objects/liferay-portal/assets/61854510/08549dae-978f-44b5-8ce1-41c8c2c13af6